packages/react-vapor/src/components/validation/ValidationIds.ts

```diff
@@ -0,0 +1,4 @@
+export const ValidationIds = {
```

Comment:
I think I would prefer ValidationTypes (and update the actions to use `type` instead of `subId`). It would be clearer

Reply:
I'm not sure about that, you would have:

```
{ type: 'UPDATE_WARNING', payload: { id: 'x', value: 'x', type: 'something' }}
```

instead of

```
{ type: 'UPDATE_WARNING', payload: { id: 'x', value: 'x', subId: 'something' }}
```

Seems a little less clear what "type" is about to me since we already have `type` in the action 🤔

What about `componentType:` or `validationType:` ?

Reply:
You would trigger ```typescript
ValidationActions.setWarning('x', 'some empty warning', ValidationTypes.nonEmpty)
```
which would translate to ```typescript
{ type: 'UPDATE_WARNING', payload: { id: 'x', value: 'some empty warning', type: 'non-empty' }}
```
It seems a little bit more easy to understand for the developper IMO

I do feel fine with either of your suggestions,  I just think subId is a bit confusing